app/analysis/template.hbs

```diff
@@ -0,0 +1,255 @@
+              prompt    = 'All cost centers'
+              options   = costCenters
+              on-change = (action (mut costCenter))
+            }}
+          {{/fs.label}}
+        {{/fs.group}}
+        {{#fs.group label='Time range'}}
+          {{#fs.label}}
+            From
+            {{fs.filter 'date'
+              data-test-filter-from-date
+              selected  = fromDate
+              on-change = (action (mut fromDate))
+            }}
+          {{/fs.label}}
+          {{#fs.label}}
+            To
+            {{fs.filter 'date'
+              data-test-filter-to-date
+              selected  = toDate
+              on-change = (action (mut toDate))
+            }}
+          {{/fs.label}}
+        {{/fs.group}}
+        {{#fs.group label='State'}}
+          {{#fs.label}}
+            Review
+            {{fs.filter 'button'
+              data-test-filter-review
+              selected  = review
+              valuePath = 'value'
+              labelPath = 'label'
+              options = (array
+                (hash label='All' value='')
+                (hash label='Review needed' value='1')
+                (hash label='No review needed' value='0')
+              )
+              on-change = (action (mut review))
+            }}
+          {{/fs.label}}
+          {{#fs.label}}
+            Billability
+            {{fs.filter 'button'
+              data-test-filter-not-billable
+              selected  = notBillable
+              valuePath = 'value'
+              labelPath = 'label'
+              options = (array
+                (hash label='All' value='')
+                (hash label='Billable' value='0')
+                (hash label='Unbillable' value='1')
+              )
+              on-change = (action (mut notBillable))
+            }}
+          {{/fs.label}}
+          {{#fs.label}}
+            Verified
+            {{fs.filter 'button'
+              data-test-filter-not-verified
+              selected  = notVerified
+              valuePath = 'value'
+              labelPath = 'label'
+              options = (array
+                (hash label='All' value='')
+                (hash label='Verified' value='0')
+                (hash label='Not verified' value='1')
+              )
+              on-change = (action (mut notVerified))
+            }}
+          {{/fs.label}}
+        {{/fs.group}}
+      {{/filter-sidebar}}
+    {{/task-selection}}
+  </div>
+
+  <div class="filter-page-content">
+    {{#if (and (not _dataCache.length) data.isRunning)}}
+      <div class="empty">
+        {{loading-icon}}
+      </div>
+    {{else}}
+      {{#with data.lastSuccessful.value as |reports|}}
+        <table class="table table--striped table--analysis">
+          <colgroup>
+            <col>
+            <col>
+            <col>
+            <col>
+            <col>
+            <col>
+            <col>
+            <col>
+            <col>
+            <col>
+          </colgroup>
```

Comment:
For what do you need the colgroup?

Reply:
`colgroup` and `col` is needed for defining column widths in a table with `table-layout: fixed;`

Reply:
alright, thanks!